README.md

```diff
@@ -64,31 +64,57 @@ several versions in a given commit, ordered by their automatic version numbers.
 
 </details>
 
-### Promoting
+### Create a label
```

Comment:
I guess i didn't make the intention clear: you can have any number of versions active under a label. The other question is that `gto show` shows only the greatest by default to not overload the screen (the same as it was with earlier - if 2 versions are both in "prod", you'll the greatest semver). There should be a param to control this, like `gto show -n 4` would show you last 4 versions with the label.

Reply:
Also, since we have "Labels" in `artifacts.yaml` already, I'm considering keeping "Stages" name. WDYT?

Reply:
No strong opinion on the name actually, but it seems like there is another possible workflow to consider (again, doesn't need to block this PR). I imagine a typical workflow looks like:

1. Put v1 in dev and deploy to dev env.
2. Move v1 to prod and deploy to prod env (in addition to dev env).
3. Move v2 to dev and deploy to dev env (replacing v1).

When I deploy v2 to the dev env, I no longer want v1 to show up in dev because it has been replaced.

Reply:
Yes, the workflow is valid. Then, I think you either:
1. use `-n 1` flag when GTO outputs multiple versions for a single env (which happens by default now in `gto show`) (which may be OK if you don't need another git tag created for some reason)
2. manually unassign the label with `gto unassign` (which is necessary if you need another git tag created for this unassignment).